src/javascript-node-mongo/devcontainer-template.json

```diff
@@ -11,6 +11,7 @@
             "type": "string",
             "description": "Node.js version (use -bullseye variants on local arm64/Apple Silicon):",
             "proposals": [
+				"22-bookworm",
```

Comment:
Looks like we missed adding `20-bookworm`, can you add as well?

Reply:
done